repology/parsers/parsers/pypi.py

```diff
@@ -90,7 +90,8 @@ def iter_parse(self, path: str, factory: PackageFactory) -> Iterable[PackageMake
 
                 if info['project_urls']:
                     for key, url in info['project_urls'].items():
-                        if (link_type := _url_types.get(key.lower())) and url != 'UNKNOWN':
+                        link_type = _url_types.get(key.lower())
```

Comment:
You can still check in the same expression. Also please check with `is not None`.